SHIBBOLETH_SP.md

```diff
@@ -6,27 +6,45 @@ category: authentication
 
 This document describes installing Shibboleth on Debian 9.
 
+## Requirements
+
+1. Apache
+
+        Apache webserver is required for Shibboleth SP.
+
+2. Root Access
+
+        It should be possible to execute commands as a root user. Please ensure you have root privileges on the system.
+
 ### Shibboleth
 
     $ sudo apt-get install libapache2-mod-shib2
     $ sudo shib-keygen
 
+The service provider should now be installed. Here are some important files and directories that may help in debugging and configuring shibboleth.
```

Comment:
This _could_ be interesting, but a little more explanation on *how* to debug / configure Shib and/or view logs would be interesting I guess... For example `tail -f` or similar commands could be provided... Or maybe there are some online resources to consult as well...

Reply:
i have editted it. i have added tail -f as an example. and also the tutorial.
For configuration shib. This part is only for information about files and directories that may help.
if i put the configuration example here. it may get redundant and may confuse the user. Because configuring shibboleth is explained later in document.